testing/integration/escaping_secrets_in_policy_test.go

```diff
@@ -30,6 +30,7 @@ func TestEscapingSecretsInPolicy(t *testing.T) {
 		Stack: &define.Stack{},
 		Sudo:  true,
 	})
+	t.Skip("flaky test: https://github.com/elastic/elastic-agent/issues/6107")
```

Comment:
Can you move this line as the first line in the test? Thanks.

Reply:
Linting starts complaining, is there a way to turn linting off for that block of code? something like `//skip-linting-here`

Reply:
There's the `nolint` tag, but don't know which one corresponds to this one

Reply:
What does it complain about?

Reply:
@swiatekm our linter enforces that the setup of the test `define.Require` is always the first line in a test. `t.Skip` should go after.